pkg/controllers/managementuser/snapshotbackpopulate/snapshotbackpopulate.go

```diff
@@ -60,6 +64,17 @@ func Register(ctx context.Context, userContext *config.UserContext) {
 	}
 
 	userContext.Core.ConfigMaps("kube-system").Controller().AddHandler(ctx, "snapshotbackpopulate", h.OnChange)
+	relatedresource.Watch(ctx, "snapshot-reconcile-trigger", func(namespace, name string, obj runtime.Object) ([]relatedresource.Key, error) {
+		logrus.Tracef("[snapshotbackpopulate] triggering reconcile on object %s/%s - cluster: %s configmap: %s", namespace, name, h.v1ClusterName, h.activeConfigMap)
```

Comment:
I would prefer that this is debug level. This is called by the wrangler controller only, so I don't know how much including the trace would benefit here.

Reply:
My understanding with `logrus.Tracef` is that it only prints when the log level is at `trace`. That being said, I don't think it's actually even useful to have this here so I'm going to remove it.